drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -3970,7 +3974,8 @@ void RasterizerSceneGLES3::render_scene(const Transform& p_cam_transform,const C
 		_render_mrts(env,p_cam_projection);
 	}
 
-	glPolygonMode(GL_FRONT_AND_BACK,GL_FILL);
+// this is not supported on iOS ES3 anymore.. 
+//	glPolygonMode(GL_FRONT_AND_BACK,GL_FILL);
```

Comment:
I guess just commenting out something that might be needed is not the right solution, is it? :D